english/wafflefries/wafflefries.md

```diff
@@ -0,0 +1,4 @@
+You sit down at the local bar and find some fries, the fries are given
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.